### PR TITLE
Stylesheet update, apply to BOLT 0, 2, 3

### PR DESCRIPTION
**Pull request by toadlyBroodle**
-add header weight, abbreviations, and 'local/remote' terminology rules to stylesheet
-apply updated stylesheet rules to BOLT 0, 2, and 3

### CODE REVIEW

File: 03-transactions.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# BOLT #3: Bitcoin Transaction and Script Formats
+a.k.a.# BOLT #3: Bitcoin Transaction and Script Formats
```
Comment (rustyrussell) [inline 03-transactions.md]: Oops :)

Comment (toadlyBroodle) [reply 03-transactions.md]: whoops, thanks ;)

**Comment by rustyrussell** [comment]
Great, am waiting for @shannona review.

File: 00-introduction.md [hunk 2]
```diff
@@ -28,9 +28,8 @@ This is version 0.
 * *Node*:
    * A computer or other device connected to the Bitcoin network.
 
-
 * *Peers*:
-   * *Nodes* wanting to transact Bitcoins with each other through a *channel*.
+   * *Nodes* transacting bitcoins with one other through a *channel*.
```
Comment (shannona) [inline 00-introduction.md]: one another?

File: 03-transactions.md [hunk 3]
```diff
@@ -518,9 +518,9 @@ In binary, it's helpful to think of any index in terms of a *prefix*,
 followed by some trailing 0s. You can derive the secret for any
 index which matches this *prefix*.
 
-For example, secret `0xFFFFFFFFFFF0` allows us to derive secrets for
-`0xFFFFFFFFFFF1` through `0xFFFFFFFFFFFF` inclusive. Secret `0xFFFFFFFFFF08`
-allows us to derive secrets `0xFFFFFFFFFF09` through `0xFFFFFFFFFF0F`
+For example, secret `0xFFFFFFFFFFF0` allows the secrets to be derived for
+`0xFFFFFFFFFFF1` through `0xFFFFFFFFFFFF`, inclusive; and secret `0xFFFFFFFFFF08`
+allows the secrets to be derive for `0xFFFFFFFFFF09` through `0xFFFFFFFFFF0F`,
```
Comment (shannona) [inline 03-transactions.md]: derived

**Comment by shannona** [comment]
Done :).

**Comment by rustyrussell** [comment]
@shannona 's points were trivial enough that I just pushed them on the end, and am applying the lot.  

I'm impatient, because this patchset contains the addition of `bitcoins` to the wordlist: currently every other patchset is marked "broken" until we merge this to fix it!

**Pull request merged**